networking/multiple_networks/primary_networks/about-user-defined-networks.adoc

```diff
@@ -22,10 +22,14 @@ image::527-OpenShift-UDN-isolation-012025.png[Namespace isolation concept in a u
 Support for the Localnet topology on both primary and secondary networks will be added in a future version of {product-title}.
 ====
 
-Unlike a network attachment definition (NAD), which is only namespaced scope, a cluster administrator can use a UDN to create and define additional networks that span multiple namespaces at the cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR). Additionally, a cluster administrator or a cluster user can use a UDN to define additional networks at the namespace level with the `UserDefinedNetwork` CR. 
+Unlike a network attachment definition (NAD), which is only namespaced scope, a cluster administrator can use a UDN to create and define additional networks that span multiple namespaces at the cluster level by leveraging the `ClusterUserDefinedNetwork` custom resource (CR). Additionally, a cluster administrator or a cluster user can use a UDN to define additional networks at the namespace level with the `UserDefinedNetwork` CR.
 
 The following sections further emphasize the benefits and limitations of user-defined networks, the best practices when creating a `ClusterUserDefinedNetwork` or `UserDefinedNetwork` custom resource, how to create the custom resource, and additional configuration details that might be relevant to your deployment.
 
+[WARNING]
+====
+A brief interruption occurs when upgrading clusters with user-defined networks enabled in 4.17.
```

Comment:
wrong version?

Reply:
I put 4.17 because you would have to enable it on 4.17 because it is TP. I thought this was what the upgrade issue was coming from. If I'm misunderstanding this, please let me know.

Reply:
You can remove this whole line. We think we can avoid the disruption now.